plugins/ltac/g_tactic.mlg

```diff
@@ -379,10 +379,12 @@ GRAMMAR EXTEND Gram
           { {onhyps=None; concl_occs=occs} }
       | "*"; "|-"; occs=concl_occ ->
           { {onhyps=None; concl_occs=occs} }
-      | hl=LIST0 hypident_occ SEP","; "|-"; occs=concl_occ ->
+      | "|-"; occs=concl_occ ->
+          { {onhyps=Some []; concl_occs=occs} }
+      | hl = LIST0 hypident_occ SEP ","; "|-"; occs=concl_occ ->
```

Comment:
And LIST1 here as well since the previous production handles the empty list?  Though I think splitting the old rule into 2 isn't necessary.  (Hope we're not going to go in circles on this detail.)

Reply:
Using `LIST0` and `LIST1` would not be factorized (I don't think that caml5 does the factorization itself, even if it could, to the price of a less readable grammar), but it seems that I indeed got confused and mixed two approaches. The previous empty list production should be enough to allow `LIST1` in the next two productions. Retrying...